bin/varnishtest/tests/r03940.vtc

```diff
@@ -0,0 +1,45 @@
+varnishtest "test startup_timeout vs. stevedore init / open"
+
+# we test with vtc_varnish and vtc_process because of different code
+# paths in mgr for implicit start vs. cli start
+
+####
+# startup_timeout used, delay in stevedore init
+varnish v1 -arg "-sdebug=debug,dinit=5s -pstartup_timeout=3s -pcli_timeout=2s" \
+	-arg "-p feature=+no_coredump" \
+	-vcl "backend none none;"
+varnish v1 -cliexpect \
+	"Child failed on launch within startup_timeout=3.00s" \
+	"start"
+varnish v1 -expectexit 0x40
+
+process p1 { varnishd \
+	-sdebug=debug,dinit=5s \
+	-pstartup_timeout=3s -pcli_timeout=2s \
+	-n ${tmpdir}/p1 -a :0 -b none 2>&1 } -start
+process p1 -expect-exit 0x2 -wait
+process p1 -expect-text 0 0 \
+	"Child failed on launch within startup_timeout=3.00s"
```

Comment:
I took @bsdphk s word that the process facility should take are.
Also, as explained in a comment, I deliberately want to test the implicit startup. `-F` is not the standard case, so would want to avoid it. 

https://github.com/varnishcache/varnish-cache/pull/3941/files/312fb5653bae2842413940312fdd58a09375bb1d#diff-98f28fac5e01e257112d0958130e8224805ee38b6f848e5ee8dc2532166c3db0R4